model/cluster_utility.go

```diff
@@ -16,6 +16,15 @@ const (
 	FLUENTBIT = "fluentbit"
 )
 
+const (
+	// PROMETHEUS_DEFAULT_VERSION defines the default version for the Helm chart
+	PROMETHEUS_DEFAULT_VERSION = "10.4.0"
+	// NGINX_DEFAULT_VERSION defines the default version for the Helm chart
+	NGINX_DEFAULT_VERSION = "1.30.0"
+	// FLUENTBIT_DEFAULT_VERSION defines the default version for the Helm chart
+	FLUENTBIT_DEFAULT_VERSION = "2.8.7"
```

Comment:
I was going to say that I was surprised that the original utility constant names passed linting, but they did so I didn't mention it.

It looks like the underscores are setting off the ALL_CAPS errors now. For our other constants we use camel casing so I would probably standardize on that.

Reply:
yeah haha I'm fixing that right now

Reply:
sorry about that!